### PR TITLE
Add a timestamp to the bottom right corner of the map

### PR DESCRIPTION
**Pull request by pleasantone**
Does what it says, very useful for screenshots.

### CODE REVIEW

File: plugins/map-timestamp.user.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  $('#updatestatus').prepend('<div id="timestamp" style="padding: 8px;"></div>');
+
+  window.plugin.mapTimestamp.update();
+};
+
+window.plugin.mapTimestamp.update = function() {
+
+  var html = 'Time: ' + new Date();
+
+  $('#timestamp').html(html);
```
Comment (dingram) [inline plugins/map-timestamp.user.js]: This could be `.text()` instead, as there is no HTML. Best to be safe :smiley:

**Review by dingram — CHANGES_REQUESTED** [review]

**Comment by pleasantone** [comment]
Requested changes made.

File: plugins/map-timestamp.user.js (hunk 1 above)
Comment (pleasantone) [inline plugins/map-timestamp.user.js]: totally agree, changes made

**Comment by pleasantone** [comment]
@dingram poke

**Pull request closed without merging**